### PR TITLE
Add a description about revoking customer's OAuth tokens

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) adds a description about revoking customer's OAuth tokens

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/customers/account-create.html

### CODE REVIEW

File: src/customers/account-create.md [hunk 1]
```diff
@@ -150,7 +150,20 @@ Customer accounts created from the Admin do not initially have passwords assigne
 
 1. In the menu bar at the top of the page, click **Reset Password**.
 
-1. Notification is sent to the account owner, with instructions for setting the password.
+   The number of password reset requests that are allowed within an hour is set in the [configuration]({% link configuration/customers/customer-configuration.md %}).
+
+## Revoke a customer’s OAuth tokens
+
+{:.bs-callout-warning}
+Do not proceed unless you are a developer familiar with API Authentication.
```
Comment (dobooth) [inline src/customers/account-create.md:158]: Why is this warning necessary?

Comment (EvheniiaKolkhiieva) [reply src/customers/account-create.md:158]: Hi @dobooth 😊 thank you for reviewing!

I have noticed, the same warning on the resetting password topic, and added to this one as well. If you think that this is mistaken, let's remove this for both topics, so as not to confuse users studying documentation from different angles
image

Comment (dobooth) [reply src/customers/account-create.md:158]: Let's remove the warning. Most users here will not be devs with API Auth experience but we don't say why that is needed. In most cases, the user will just then login again as usual. No need to touch the password file.

**Review by dobooth — CHANGES_REQUESTED** [review]

**Comment by EvheniiaKolkhiieva** [comment]
Done @dobooth, please check 😊
Thank you!

**Comment by keharper** [comment]
I'm closing this PR, because the act of forcing a log in is outside the scope of the topic, which is about creating customer accounts.

**Pull request closed without merging**